History.md

```diff
@@ -6,6 +6,8 @@ v2.0.0 / 2017-07-13
   * Add support for performance.now (#106)
   * Fix issue with tick(): setSystemClock then throw
   * Update old dependencies
+  * Added support to automatically increment time (#85)
```

Comment:
that's very considerate, but you can drop this. we create this semi-automatically when crafting a new release using `git-extras`